similarity-data-api/src/main/java/org/phenotips/data/similarity/PatientGenotype.java

```diff
@@ -0,0 +1,55 @@
+/*
+ * See the NOTICE file distributed with this work for additional
+ * information regarding copyright ownership.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+package org.phenotips.data.similarity;
+
+import org.xwiki.stability.Unstable;
+
+import java.util.Set;
+
+/**
+ * This class represents the combined genetics (candidate genes and exome sequence data) for a Patient.
+ *
+ * @version $Id$
+ * @since 1.0M6
+ */
+@Unstable
+public interface PatientGenotype extends Exome
+{
+    /**
+     * Return whether the patient has any genotype data available (candidate genes or exome data).
```

Comment:
Terminology: does `candidate` mean all recorded genes, candidate or causative? Should we remove `candidate`, or replace it with something else?